roles/accprovision/manifests/net-operator-controller-deployment.yaml

```diff
@@ -0,0 +1,416 @@
+                type: boolean
+            type: object
+        required:
+        - spec
+        type: object
+---
+apiVersion: v1
+data:
+  spec: "{\n    \"acc_provision_input\": {\n        \"operator_managed_config\": {\n\
+    \            \"enable_updates\": false\n        },\n        \"aci_config\": {\n\
+    \            \"system_id\": \"mykube\",\n            \"apic_hosts\": [\n     \
+    \           \"10.1.1.101\"\n            ],\n            \"aep\": \"kube-cluster\"\
+    ,\n            \"tenant\": {\n                \"name\": \"prj_5cc3c29ec36a49cf942ff84f9e8deb30\"\
+    \n            },\n            \"vrf\": {\n                \"name\": \"mykube-vrf\"\
+    ,\n                \"tenant\": \"common\"\n            },\n            \"l3out\"\
+    : {\n                \"name\": \"mykube_l3out\",\n                \"external_networks\"\
+    : [\n                    \"mykube_extepg\"\n                ]\n            }\n\
+    \        },\n        \"registry\": {\n            \"aci_containers_controller_version\"\
+    : \"kmr2-test\", \n            \"aci_containers_host_version\": \"kmr2-test\"\
+    , \n            \"aci_containers_operator_version\": \"kmr2-test\", \n       \
+    \     \"cnideploy_version\": \"kmr2-test\", \n            \"image_prefix\": \"\
+    quay.io/noirolabs\", \n            \"network_operator_version\": \"test\", \n\
+    \            \"openvswitch_version\": \"kmr2-test\", \n            \"opflex_agent_version\"\
+    : \"kmr2-test\"\n        },\n        \"net_config\": {\n            \"infra_vlan\"\
+    : 3801,\n            \"kubeapi_vlan\": 2032,\n\
+    \            \"extern_dynamic\"\
+    : \"dummy\",\n\
+    \            \"node_subnet\": \"dummy\",\n            \"pod_subnet\": \"\
+    dummy\"\n        }\n    }\n }"
+kind: ConfigMap
+metadata:
+  name: acc-provision-config
+  namespace: aci-containers-system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: net-operator-controller
+  namespace: aci-containers-system
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      name: net-operator-controller
+      network-plugin: aci-containers
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+      labels:
+        name: net-operator-controller
+        network-plugin: aci-containers
+      name: net-operator-controller
+      namespace: aci-containers-system
+    spec:
+      containers:
+      - env:
+        - name: ANSIBLE_GATHERING
+          value: explicit
+        - name: WATCH_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: ACC_PROVISION_FLAVOR
+          value: cko-calico
+        - name: ACC_PROVISION_INPUT_CR_NAME
+          value: accprovisioninput
+        image: quay.io/noirolabs/acc-provision-operator:abhishek
+        imagePullPolicy: Always
```

Comment:
Can we change this to IfNotPresent?

Reply:
yes. But this is in reference to every other container image, which is Always, so that it takes the updated image automatically whenever new image is pushed and we delete the pods.